### DISCUSSION TITLE
member page: member's paper button

### DISCUSSION THREAD

**Discussion by jcole150** [greenelab/lab-website-template#143] [Q&A] [answered]
I don't think it's super clear at first that the "[member]'s papers" button on the bottom of each member page shows only papers that overlap with the PI/lab. It makes new members of the lab look unpublished.

I'm not sure what the best fix for this is. The only things I came up with were the following non-ideal suggestions. 
- rename to "[member]'s papers in the X lab." (though that may look funny for the PI)
- have the option to remove the button until they publish together. 

not a big deal - but just curious if there were any better potential solutions.

**Comment by vincerubinetti** [comment, marked as answer]
The button, by default, just links to the "research" page with the member's name in the url so that whatever is listed on the research page gets filtered.

So I think the button's text should be updated to say "X's papers on the Research page" to be accurate. You can also remove the `style="button"` to give it less visual emphasis.

Presumably, to emphasize a member's full published works, you'd want to makes sure they add their ORCID, Google Scholar, etc. to their bio file front matters, which show up as prominent links right under their picture.

**Reply by vincerubinetti** [reply]
I've made a hot fix here https://github.com/greenelab/lab-website-template/commit/3386f78babca66f7b3d65e19647a589f48c13f63
